MeetupPlatform.Api/Features/Meetups/LeftReplyComment/LeftReplyCommentFeature.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace MeetupPlatform.Api.Features.Meetups.LeftReplyComment;
+
+using AutoMapper;
+using MeetupPlatform.Api.Domain.Comments;
+using MeetupPlatform.Api.Persistence.Context;
+using MeetupPlatform.Api.Seedwork.WebApi;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Meetups)]
+public class LeftReplyCommentFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
+
+    public LeftReplyCommentFeature(ApplicationContext context, IMapper mapper)
+    {
+        this.context = context;
+        this.mapper = mapper;
+    }
+
+    /// <summary>
+    /// Left reply to root comment.
+    /// </summary>
+    /// <response code="404">If needed meetup is null or needed comment doesn't exist.</response>
+    /// <response code="201">Returns the new created item.</response>
+    [HttpPost("/api/meetups/{meetupId:guid}/comments/{commentId:guid}/replies")]
+    [Authorize]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(typeof(CreatedReplyDto), StatusCodes.Status201Created)]
+    public async Task<IActionResult> LeftReplyComment(
+        [FromRoute] Guid meetupId, 
+        [FromRoute] Guid commentId, 
+        [FromBody] CreationReplyDto creationReplyDto)
+    {
+        var meetup = await context.Meetups
+            .Include(meetup => meetup.Comments)
+            .Include(meetup => meetup.SignedUpUsers)
+            .Where(meetup => meetup.Id == meetupId)
+            .SingleOrDefaultAsync();
+        if(meetup is null)
+        {
+            return NotFound();
+        }
+
+        bool isCommentExistInMeetup = meetup.Comments.Any(rootComment => rootComment.Id == commentId);
```

Comment:
As we aren't using data inside the `Comments` navigation property, we can either:
1. Execute a separate DB call checking if this comment exists on the specified meetup (the call will be a lot more efficient that loading all comments from the DB)
2. Use the comment instance itself and it's `ReplyComments` navigation property to left a reply comment